### DISCUSSION TITLE
Shutter with single pulse?

### DISCUSSION THREAD

**Discussion by buzzer8** [arendst/Tasmota#19155] [General]
Hi!
I already tried everything, which i could find online, but i failed. 

I use Tasmota for my garage door. Therefore i switch 12V wires (1 for up, 1 for down) to ground, coming from the garage door motor circuit, normaly connected to a wall switch. I need a pulse, but if i use pulsetime1 5 and pulsetime2 5 i get a pulse a start AND at stop. Is there a way to only get one pulse at start? 

Thank you!

**Comment by Lupla** [comment]
I have exactly the same problem.
Although I don't need this for my garage door. I need this for my normal shutters at home, which have an endpoint and don't need the extra stopping signal. I control a shutter motor via 2 relays which only have to puls a button inside the motor. The motor is not integrated into the shutter. It is such a thing: https://amzn.to/4sIztsF
Did you find a solution for this?
